cli/state.go

```diff
@@ -1872,3 +1878,82 @@ var StateNtwkVersionCmd = &cli.Command{
 		return nil
 	},
 }
+
+var StateSysActorCIDsCmd = &cli.Command{
+	Name:  "actor-cids",
+	Usage: "Returns the built-in actor bundle manifest ID & system actor cids",
+	Flags: []cli.Flag{
+		&cli.UintFlag{
+			Name:  "network-version",
+			Usage: "specify network version",
+			Value: uint(build.NewestNetworkVersion),
+		},
+	},
+	Action: func(cctx *cli.Context) error {
+		if cctx.Args().Present() {
+			return ShowHelp(cctx, fmt.Errorf("doesn't expect any arguments"))
+		}
+
+		api, closer, err := GetFullNodeAPI(cctx)
+		if err != nil {
+			return err
+		}
+		defer closer()
+
+		ctx := ReqContext(cctx)
+
+		ts, err := LoadTipSet(ctx, cctx, api)
+		if err != nil {
+			return err
+		}
+
+		nv, err := api.StateNetworkVersion(ctx, ts.Key())
+		if err != nil {
+			return err
+		}
+
+		if cctx.IsSet("network-version") {
+			nv = network.Version(cctx.Uint64("network-version"))
+		}
+
+		fmt.Printf("Network Version: %d\n", nv)
+
+		actorVersion, err := actors.VersionForNetwork(nv)
+		if err != nil {
+			return err
+		}
+		fmt.Printf("Actor Version: %d\n", actorVersion)
+
+		manifestCid, ok := actors.GetManifest(actorVersion)
+		if !ok {
+			return xerrors.Errorf("cannot get manifest CID")
+		}
+		fmt.Printf("Manifest CID: %v\n", manifestCid)
+
+		tw := tabwriter.NewWriter(os.Stdout, 2, 4, 2, ' ', 0)
+		_, _ = fmt.Fprintln(tw, "\nActor\tCID\t")
+
+		for _, name := range []string{
+			actors.AccountKey,
```

Comment:
Can I bug you to make one refactor? Add:

```
func GetBuiltinActorsKeys() []string {
	return []string{
		AccountKey,
		CronKey,
		InitKey,
		MarketKey,
		MinerKey,
		MultisigKey,
		PaychKey,
		PowerKey,
		RewardKey,
		SystemKey,
		VerifregKey,
	}
}
```

to chain/actors/manifest.go, and use that here (and in chain/actors/manifest.go).